lib/Pusher.php

```diff
@@ -258,8 +269,10 @@ private function validate_socket_id($socket_id)
     /**
      * Utility function used to create the curl object with common settings.
      */
-    private function create_curl($s_url, $request_method = 'GET', $query_params = array())
+    private function create_curl($s_url, $request_method = 'GET', $query_params = array(), $notification_host = false)
```

Comment:
Fair. I didn't want to change things much, but that sounds like a much much better way.